api/src/api/elevatorSafetyRouter.ts

```diff
@@ -18,5 +22,18 @@ export const elevatorSafetyRouterFactory = (
       });
   });
 
+  router.post("/elevator-safety/registration/", async (req, res) => {
+    const { address } = req.body;
```

Comment:
Could we do this and drop line 27?
```suggestion
    const { address, zipCode } = req.body;
```